### PR TITLE
fix(#1): remove all but one "restart open-xchange" handlers to prevent multiple and uncontrolled restarts

### PR DESCRIPTION
**Pull request by s3lph**

### CODE REVIEW

File: roles/clientonboarding/tasks/config.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     owner: root
     group: root
     mode: "0644"
-  notify: restart open-xchange
+  notify: Restart Open-Xchange
```
Comment (Daenou) [inline roles/clientonboarding/tasks/config.yml:10]: Missing ""

File: roles/documentconverter/tasks/config.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@
     group: root
     mode: "0644"
   notify:
-    - restart open-xchange
+    - Restart Open-Xchange
```
Comment (Daenou) [inline roles/documentconverter/tasks/config.yml:11]: Not sure if quotes are needed in a list but adding a comment anyway.

File: roles/documentconverter/tasks/install.yml [hunk 3]
```diff
@@ -25,5 +25,5 @@
       - open-xchange-documentconverter-client
       - readerengine
   notify:
-    - restart open-xchange
+    - Restart Open-Xchange
```
Comment (Daenou) [inline roles/documentconverter/tasks/install.yml:28]: Not sure if quotes are needed in a list but adding a comment anyway

File: roles/dynamictheme/tasks/config.yml [hunk 4]
```diff
@@ -25,7 +25,7 @@
     group: root
     mode: "0644"
   notify:
-    - restart open-xchange
+    - Restart Open-Xchange
```
Comment (Daenou) [inline roles/dynamictheme/tasks/config.yml:28]: Not sure if quotes are needed in a list but adding a comment anyway

File: roles/dynamictheme/tasks/install.yml [hunk 5]
```diff
@@ -5,4 +5,4 @@
     name:
       - open-xchange-dynamic-theme
   notify:
-    - restart open-xchange
+    - Restart Open-Xchange
```
Comment (Daenou) [inline roles/dynamictheme/tasks/install.yml:8]: Not sure if quotes are needed in a list but adding a comment anyway

File: roles/mobileapifacade/tasks/install.yml [hunk 6]
```diff
@@ -12,4 +12,4 @@
       - open-xchange-mobile-api-facade
       - open-xchange-pns-mobile-api-facade
   notify:
-    - restart open-xchange
+    - Restart Open-Xchange
```
Comment (Daenou) [inline roles/mobileapifacade/tasks/install.yml:15]: Not sure if quotes are needed in a list but adding a comment anyway

File: roles/mobilityconnector/tasks/config.yml [hunk 7]
```diff
@@ -8,7 +8,7 @@
     group: root
     mode: "0644"
   notify:
-    - restart open-xchange
+    - Restart Open-Xchange
```
Comment (Daenou) [inline roles/mobilityconnector/tasks/config.yml:11]: Not sure if quotes are needed in a list but adding a comment anyway

File: roles/mobilityconnector/tasks/install.yml [hunk 8]
```diff
@@ -19,5 +19,5 @@
     name:
       - open-xchange-meta-mobility
   notify:
-    - restart open-xchange
+    - Restart Open-Xchange
```
Comment (Daenou) [inline roles/mobilityconnector/tasks/install.yml:22]: Not sure if quotes are needed in a list but adding a comment anyway

File: roles/owncloud/tasks/config.yml [hunk 9]
```diff
@@ -8,7 +8,7 @@
     group: root
     mode: "0644"
   notify:
-    - restart open-xchange
+    - Restart Open-Xchange
```
Comment (Daenou) [inline roles/owncloud/tasks/config.yml:11]: Not sure if quotes are needed in a list but adding a comment anyway

File: roles/owncloud/tasks/install.yml [hunk 10]
```diff
@@ -14,4 +14,4 @@
       - open-xchange-owncloud-filepicker
       - open-xchange-owncloud-oauth-session
   notify:
-    - restart open-xchange
+    - Restart Open-Xchange
```
Comment (Daenou) [inline roles/owncloud/tasks/install.yml:17]: Not sure if quotes are needed in a list but adding a comment anyway

**Review by Daenou — DISMISSED** [review]
Some inconsistencies around quoting, otherwise all good

**Comment by s3lph** [comment]
Rather than adding quotes everywhere, I removed them in the few places I had added them, as they are not necessary anymore.

**Review by Daenou — APPROVED** [review]
LGTM!

**Comment by s3lph** [comment]
Released as v1.0.3

**Pull request merged**